src/Command/GenerateCommand.php

```diff
@@ -161,17 +168,31 @@ private function copyStaticPhpFiles(StyleInterface $ss): void
 
     private function copySpecification(StyleInterface $ss): void
     {
-        $ss->text(sprintf('Copy specification file to %s.', $this->configuration->getOutputDirectory()));
-
         $destinationPath = sprintf(
             '%s/doc/%s',
             $this->configuration->getOutputDirectory(),
             basename($this->configuration->getSpecificationFilePath())
         );
 
+        $ss->text(sprintf('Copy specification file to %s.', $destinationPath));
+
         $this->filesystem->copy(
             $this->configuration->getSpecificationFilePath(),
             $destinationPath
         );
     }
+
+    private function initWarningPrinting(InputInterface $input): void
+    {
+        if ($input->getOption('quiet')) {
+            set_error_handler(
+                static function (): bool {
+                    return true;
+                },
+                E_USER_WARNING
+            );
+        } else {
+            set_error_handler($this->warningFormatter);
```

Comment:
```suggestion
            set_error_handler($this->warningFormatter, E_USER_WARNING);
```
otherwise when generating the client LOTS of yellow messages are popping up 🤣